### DISCUSSION TITLE
Deprecated ToggleGroup component

### DISCUSSION THREAD

**Discussion by matandre1** [telekom/scale#1576] [Q&A] [answered]
Hello,

I would like to ask you if _ToggleGroup_ component is completely deprecated or only property _colorScheme_ is deprecated as documentation shown.

So it is safe to use _ToggleGroup_ without this property or should I use _SegmentedButton_ component instead?

Thank You.

**Comment by acstll** [comment, marked as answer]
Hey, sorry for the late reply!

The entire component Toggle Group is deprecated. The new Segmented Button should replace it, so it's recommended to use that instead.
